app/helpers/series_helper.rb

```diff
@@ -1,28 +1,28 @@
-module SeriesHelper 
-  
+module SeriesHelper
+
   def show_series_data(work)
     series_data = series_data_for_work(work)
     series_data.join(ArchiveConfig.DELIMITER_FOR_OUTPUT).html_safe
   end
-  
+
   # this should only show prev and next works visible to the current user
   def series_data_for_work(work)
     series = work.series.select{|s| s.visible?(current_user)}
     series.map do |serial|
       serial_works = serial.serial_works.find(:all, :include => :work, :conditions => ['works.posted = ?', true], :order => :position).select{|sw| sw.work.visible(current_user)}.collect{|sw| sw.work}
-      visible_position = serial_works.index(work) || serial_works.length     
+      visible_position = serial_works.index(work) || serial_works.length
       unless !visible_position
-        previous_link = (visible_position > 0) ? link_to("&laquo; ".html_safe, serial_works[visible_position - 1]) : "".html_safe
-        main_link = ("Part " + (visible_position+1).to_s + " of the " + link_to(serial.title, serial) + " series").html_safe
-        next_link = (visible_position < serial_works.size-1) ? link_to(" &raquo;".html_safe, serial_works[visible_position + 1]) : "".html_safe
+        previous_link = (visible_position > 0) ? link_to("&larr; Previous Work".html_safe, serial_works[visible_position - 1]) : "".html_safe
+        main_link = (" Part " + (visible_position+1).to_s + " of the " + link_to(serial.title, serial) + " series ").html_safe
```

Comment:
This raw mix of strings and variables arranged in English-language order will be impossible to translate when the day comes, particularly without being wrapped in `ts`. While you're editing this line, I'd suggest using interpolation to make it a single string, e.g.:

```ruby
main_link = ts("Part #{(visible_position + 1).to_s} of the #{link_to(serial.title, serial)} series").html_safe
```

or alternatively to make the text itself more legible:

```ruby
main_link = ts("Part %{position} of the %{series_title} series", position: (visible_position + 1).to_s, series_title: link_to(serial.title, serial)).html_safe
```

The spaces at either end feel as though they've been added for aesthetic reasons that should probably be addressed in the aesthetic layer? (i.e. HTML or CSS)